dotnet/csharp.bzl

```diff
@@ -359,7 +361,7 @@ Args:
 
 csharp_binary = rule(
     implementation = _csc_compile_impl,
-    attrs = dict(_COMMON_ATTRS.items() + _EXE_ATTRS.items()),
```

Comment:
do not revert those change, they are because dictionary addition is not python compatible so this file can be parsed as python